pom.xml

```diff
@@ -31,9 +31,11 @@ under the License.
     <modules>
         <module>flink-connector-oceanbase-base</module>
         <module>flink-connector-obkv-hbase</module>
+        <module>flink-connector-directload</module>
```

Comment:
Modules other than base should be listed in alphabetical order.

And for the module name, to keep consistent with the IDENTIFIER, maybe we also add 'oceanbase-'.

Reply:
- Reordered.
- Added `oceanbase-`